CMakeLists.txt

```diff
@@ -2,7 +2,15 @@ cmake_minimum_required(VERSION 2.8)
 
 set(PROJECT_NAME mc-mini)
 set(PROJECT_VERSION 1.1.0)
-project(${PROJECT_NAME})
+
+if(CMAKE_VERSION LESS 3.0)
```

Comment:
While this works, I'd actually suggest using the following block instead:
```cmake
if(POLICY CMP0048)
  cmake_policy(SET CMP0048 NEW)
  project(${PROJECT_NAME} VERSION ${project_version})
else()
  project(${PROJECT_NAME})
endif()
```
This is slightly easier to follow, since it puts the expected case before the fallback case, and it has the added (although probably negligible) benefit that if the ability to set policy CMP0048 magically disappears in some future version of CMake, it won't break our build.